staff/carlos-calvo/e-commerce-template/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>e-commerce</title>
+</head>
+<body>
+    <header class="header">
+        <img class = "logo" src="https://cdn4.iconfinder.com/data/icons/new-google-logo-2015/400/new-google-favicon-512.png">
+        <h3>NOM PÀGINA WEB</h3>
+        <form>
+            <input class="search" type="search" value="search">
+            <button>
+                <i class="fa fa-search"></i>
+            </button>
+        </form>
+        <div>
```

Comment:
You could refactor all that section, deleting one div tag that seems that is duplicated and also trying to put everything inside the nav tag.